homeassistant/components/devolo_home_control/strings.json

```diff
@@ -5,18 +5,18 @@
       "already_configured": "This Home Control Central Unit is already in use."
     },
     "error": {
-      "invalid_credentials": "Incorrect user name and/or password."
+      "invalid_credentials": "[%key:common::config_flow::error::invalid_auth%]"
     },
     "step": {
       "user": {
         "data": {
-          "username": "E-Mail-Address / devolo ID",
+          "username": "[%key:common::config_flow::data::email%] / devolo ID",
```

Comment:
Would it be better to change the whole key to the common one ?

Reply:
Well, if you are new to Home Assistant but know devolo Home Control well, you are used to "devolo ID". That's why I thought keeping it makes sense.

Reply:
I streamlined the reference key to email.

Reply:
You need to update the tests as well.